gdx/src/com/badlogic/gdx/graphics/g2d/GlyphLayout.java

```diff
@@ -165,7 +164,7 @@ else if (targetWidth <= fontData.spaceXadvance * 3) //
 						glyphRunPool.free(run);
 						break runEnded;
 					}
-					int n = run.xAdvances.size;
+					int xAdvancesSize = run.xAdvances.size;
```

Comment:
xAdvancesSize is easier to understand than having to remember what `n` is supposed to be all the time